docs/external-listener/index.md

```diff
@@ -36,6 +36,14 @@ listenersConfig:
       name: "external1"
       externalStartingPort: 19090
       containerPort: 9094
+      # set which port clients can use to reach all the brokers of the Kafka cluster, default is 29092
+      # valid range: 0 < x < 65536
+      # this doesn't have impact if using NodePort to expose the Kafka cluster
+      anyCastPort: 443
```

Comment:
**Optional**: We **MAY** consider using the Go style description to make it obvious which field we are talking about.

```suggestion
      # anyCastPort sets which port clients can use to reach all the brokers of the Kafka cluster, default is 29092
      # valid range: 0 < x < 65536
      # this doesn't have impact if using NodePort to expose the Kafka cluster
      anyCastPort: 443
```